### PR TITLE
HOTFIX - Minor changes

### PR DESCRIPTION
**Pull request by b-hayes**
Doc described the method as only returning a string. But it can return a bool as well.
Also, update the composer.json to include the PDO extension.

### CODE REVIEW

File: lib/PicoDb/Table.php [hunk 1]
```diff
@@ -728,14 +728,14 @@ public function offset($value)
     }
 
     /**
-     * Group by
+     * Group By
      *
-     * @access public
+     * @param ...$columns
```
Comment (joshmcrae) [inline lib/PicoDb/Table.php:733]: You can add a type for variadic arguments that IDEs should understand.

```
@param string ...$columns
```

**Review by joshmcrae — CHANGES_REQUESTED** [review]
Looks good! I just have one improvement to make to the DocBlocks.

File: lib/PicoDb/Table.php (hunk 1 above)
Comment (b-hayes) [reply lib/PicoDb/Table.php:733]: there is probably a few others that are variadic just like this one too.

**Comment by b-hayes** [comment]
Should we add return/parm type hints for everything? Or perhaps thats best as a separate upgrade.

**Comment by joshmcrae** [comment]
@b-hayes that could be a large change and we'll want to spend time making sure any added type hints don't result in type errors (e.g. because mixed types aren't documented, which is what you're fixing here).

Give the new DocBlock for `$columns` a type and we can get this merged as it is.

```
@param string ...$columns
```

**Comment by b-hayes** [comment]
Already done.

**Comment by b-hayes** [comment]
And somehow its broken now from that doc block change.
Some issue with the autoloader any ideas?

**Review by joshmcrae — APPROVED** [review]
@b-hayes deleting the action's cache for Composer and re-running the failed jobs sorted it out. I'm guessing something about the environment the actions run in has changed in a way that's incompatible with what was cached.

**Review by joshmcrae — COMMENTED** [review]
Thanks @b-hayes.

Released as [4.2.1](https://github.com/elvanto/picodb/releases/tag/4.2.1).

**Pull request merged**